dist/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Infozona</title>
+</head>
+<body>
+    <div id="app"></div>
+    <script src="./bundle.js"></script>
```

Comment:
Well I guess it doesn't matter then. Did you try with `git rm --cached dist/index.html`?